internal/util/setters/setters.go

```diff
@@ -37,14 +37,19 @@ const (
 	GcloudProjectNumber = "gcloud.project.projectNumber"
 )
 
-func PerformAutoSetters(path string) error {
+func PerformAutoSetters(targetPath, parentPath string) error {
```

Comment:
So here we basically try to set all of them and let the last win? I think it would make more sense if we try them in the correct order and then exit early if we are successful. In this current solution, if the value from env doesn't pass validation, this will fail even if the inherited value (which is the only one we care about) is valid.

Reply:
Done. Leveraged isSet flag.